extra/1-currency-conversion.js

```diff
@@ -15,7 +17,10 @@ function convertToUSD() {}
   They have also decided that they should add a 1% fee to all foreign transactions, which means you only convert 99% of the £ to BRL.
 */
 
-function convertToBRL() {}
+function convertToBRL(price) {
+  let newPrice = (price * 99) / 100;
```

Comment:
Suggestion: You could do price * 0.9;

Reply:
Thank you Kara, you have spotted every detail.I appreciate your support.